react/features/filmstrip/functions.native.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import { toState } from '../base/redux';
+
+/**
+ * Returns true if the filmstrip on mobile is visible, false otherwise.
+ *
+ * NOTE: Filmstrip on mobile behaves differently to web, and is only visible
+ * when there are at least 2 participants.
+ *
+ * @param {Object | Function} stateful - The Object or Function that can be
+ * resolved to a Redux state object with the toState function.
+ * @returns {boolean}
+ */
+export function isFilmstripVisible(stateful: Object | Function) {
+    const state = toState(stateful);
+    const { length: participantCount } = state['features/base/participants'];
```

Comment:
use getParticipantCount here. Generally, we started a transition away from checking `features/base/participants` directly, because one might be accessing stalled content.

Reply:
We've already discussed this, that gives a different result as it excludes bots. I'm okay to change this if someone confirms that bots are okay to be excluded from the count that controls the filmstrip behaviour

Reply:
I suppose that bots should be excluded from the count if they are not displayed in the filmstrip. Are they ?

Reply:
Btw. I have specifically asked to have this method introduced in a separate commit. Could you please separate it ?